includes/blocks/course-theme/class-course-content.php

```diff
@@ -96,6 +96,11 @@ public function render_content( $content ) {
 	 * @return false|string
 	 */
 	private function render_lesson_content() {
+
+		if ( ! in_the_loop() && have_posts() ) {
+			the_post();
```

Comment:
I think the purpose of this is set the correct page, right? If yes, I think it would be worth to add a comment explaining that it's here, just to make the code easier to understand. :)

Reply:
Turns out this is actually not needed for pagination. We can keep it since it's from the core `post-content` block, to make sure things are set up correctly for plugins and stuff relying on the post an in_the_loop behaving predictably.
Added comment: [ccef80d](https://github.com/Automattic/sensei/pull/4686/commits/ccef80d123b88500fc2a8665ab72fd3f4a6761aa)